micromamba/tests/conftest.py

```diff
@@ -50,11 +50,6 @@ def tmp_clean_env(
     tmp_pkgs_dirs: pathlib.Path, shared_pkgs_dirs: bool
 ) -> Generator[None, None, None]:
     """Remove all Conda/Mamba activation artifacts from environment."""
-    saved_environ = {}
-    for k, v in os.environ.items():
-        if k.startswith(("CONDA", "_CONDA", "MAMBA", "_MAMBA")):
-            saved_environ[k] = v
```

Comment:
I guess this should not be removed since it's used below in:
```
def keep_in_path(
    p: str, prefix: str | None = saved_environ.get("CONDA_PREFIX")
```

Reply:
Good catch, forgot to commit a change that changes this to `os.environ.get("CONDA_PREFIX")`